packages/app/lib/ui/widgets/sighting_form.dart

```diff
@@ -1,149 +1,143 @@
+            enableInfiniteScroll: false,
+            viewportFraction: 1,
+            padEnds: false,
+            enlargeCenterPage: true),
+        items: images.indexed.map((item) {
+          return Builder(
+            builder: (BuildContext context) {
+              int index = item.$1;
+              Image image = item.$2;
+              return CarouselItem(
+                  image: image, index: index, showDelete: showDelete);
+            },
+          );
+        }).toList(),
+      ),
+    );
+  }
+}
+
+class CarouselItem extends StatelessWidget {
+  const CarouselItem(
+      {super.key,
+      required this.image,
+      required this.index,
+      this.showDelete = false});
+
+  final Image image;
+  final int index;
+  final bool showDelete;
+
+  @override
+  Widget build(BuildContext context) {
+    final cameraImageProvider = MeliCameraProviderInherited.of(context);
+
+    return Container(
+        width: double.infinity,
+        decoration: BoxDecoration(
+            color: MeliColors.pink,
+            borderRadius: BorderRadius.all(Radius.circular(12.0)),
+            border: Border.all(width: 3, color: Colors.black)),
+        child: Stack(children: [
+          Container(
+            alignment: Alignment.center,
+            child: image,
           ),
-          Row(
-            children: <Widget>[
-              Expanded(
-                child: ElevatedButton(
-                  onPressed: () {
-                    if (_formKey.currentState?.saveAndValidate() ?? false) {
-                      debugPrint(_formKey.currentState?.value.toString());
-                    } else {
-                      debugPrint(_formKey.currentState?.value.toString());
-                      debugPrint('validation failed');
-                    }
-                  },
-                  child: const Text(
-                    'Submit',
-                    style: TextStyle(color: Colors.white),
-                  ),
-                ),
-              ),
-              const SizedBox(width: 20),
-              Expanded(
-                child: OutlinedButton(
-                  onPressed: () {
-                    _formKey.currentState?.reset();
-                  },
-                  // color: Theme.of(context).colorScheme.secondary,
-                  child: Text(
-                    'Reset',
-                    style: TextStyle(
-                        color: Theme.of(context).colorScheme.secondary),
-                  ),
+          if (showDelete)
+            Container(
+              alignment: Alignment.bottomCenter,
+              child: ElevatedButton(
+                style: ButtonStyle(
+                    backgroundColor:
+                        MaterialStateProperty.all<Color>(Colors.red[800]!),
+                    shape: MaterialStateProperty.all<CircleBorder>(
+                        CircleBorder())),
+                onPressed: () {
+                  cameraImageProvider.removeAt(index);
```

Comment:
Also here it should rather provider a generic callback that an image was deleted. The camera provider is too specific here (we will need this carousel also in other views, independent of the camera).